pdc/apps/common/renderers.py

```diff
@@ -174,6 +180,20 @@ def replace_url(match):
         return URL_SPEC_RE.sub(replace_url, text)
 
 
+def get_ordering_field(view, method):
+    """ If the APIs have the LIST method; for the view of LIST method, add the
+    Ordering field for the users.
+    """
+    if 'list' in method and view.serializer_class:
```

Comment:
Some people suggest it's better to have a explicit return if the return value is useful, E.g. http://stackoverflow.com/questions/4027586/is-it-ok-to-skip-return-none. For me it seems a little more readable. 
@simozhan what do you think?